napari/_qt/dialogs/qt_plugin_dialog.py

```diff
@@ -689,7 +689,12 @@ def tag_unavailable(self, project_info: PackageMetadata):
             widget = self.itemWidget(item)
             widget.show_warning(
                 trans._(
-                    "Plugin not yet available for installation within the bundle application"
+                    "This plugin is not available on conda-forge, so cannot be"
+                    " installed into this bundle. Ask the plugin developer to "
+                    "publish their plugin on conda-forge! Or if you use the .zip "
```

Comment:
```suggestion
                    " installed into this bundle. If you use the .zip "
```

I don't really like the idea of asking people to pester plugin developers even when it's a good recommendation.